src/components/step/StepImageIcon/StepImageIcon.css

```diff
@@ -0,0 +1,11 @@
+.step-container .step-image-icon {
```

Comment:
I unfortunately missed this on @ashleyrtate 's original PR, but I think for now at least we should prefix all styles with ```.mdhui- ``` since that's the pattern we've followed so far in this repo

Reply:
Seems pretty safe to just refactor those style names in this PR if you want. I don't think they would be referenced anywhere else yet.

Reply:
Yeah - pretty focused find/replace.  I've prefixed everything in the new "Step" components.  Can all be verified via the storybook.